nixos/modules/virtualisation/openvswitch.nix

```diff
@@ -42,6 +42,9 @@ in {
       default = false;
       description = ''
         Whether to start racoon service for openvswitch.
+        Supported only if openvswitch version is less than 2.6.0.
```

Comment:
Did they drop support in later versions?

Reply:
Yes they did: https://github.com/openvswitch/ovs/commit/2b02d770c4cb381ec32cd4b7b1e991c42b448884